src/components/Header/header.css

```diff
@@ -0,0 +1,24 @@
+.Header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    color: #f65261;
+    font-size: 25px;
+    font-family: Helvetica, serif;
+    margin-left: 50px;
+    margin-right: 50px;
+    align-items: center;
+}
+
+.AddMovieButton {
+    height: 50px;
+    width: 150px;
+    background-color: rgba(85, 85, 85, .5);
+    color: #f65261;
+    font-size: 16px;
+    border-radius: 5px;
+    text-decoration: none;
+    border-style: solid;
```

Comment:
Seems that border is not visible due to `border: 0`. Is it still needed?

Reply:
looks like it's needed, if I remove it the angles of the button start looking square